pages/common/dmesg.md

```diff
@@ -0,0 +1,11 @@
+# dmesg
+
+> Write the kernel messages in Linux and other Unix-like operating systems to standard output.
+
+- Show how much physical memory is available on this system:
```

Comment:
I think we should first put the example of the simplest command usage, which is just `dmesg`.